.gitignore

```diff
@@ -54,6 +52,9 @@ npm-debug.log
 yarn-error.log
 testem.log
 /typings
+# Build Files
+dist/
```

Comment:
This line is not needed as it is already defined here:
https://github.com/amplication/plugins/pull/353/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947R4